README.md

```diff
@@ -1,28 +1,65 @@
 # OmbuLabs::Auth
-Short description and motivation.
+> _Refactor a rack middleware (probably a ruby gem) out of https://points.ombulabs.com/ so that we can easily add GitHub-based authentication to other Ruby/Rails internal applications._
+>
+> _It would be useful if we could add it to http://skunk.fastruby.io (backend for skunk) so that anybody who is part of our organization could sign in and review details in the admin panel._
+>
+> _Said gem should be easily configurable (only with environment variables) and should make it very simple to add "OmbuLabs-authentication" to our internal Ruby apps._
 
-## Usage
-How to use my plugin.
+## Configs
+Make sure you configure your ENV variables to use Github authentication.
 
+```
+ENV["GITHUB_APP_ID"]
+ENV["GITHUB_APP_SECRET"]
+ENV["ORGANIZATION_LOGIN"]
+```
 ## Installation
-Add this line to your application's Gemfile:
+
+- Clone this gem inside the `lib` folder of your project
```

Comment:
@tiagocasanovapt We don't want to encourage people to do this ☝️

Reply:
I left these instructions for the repo **_as is_** 😅

Reply:
@tiagocasanovapt I get that, but the *as is* is going to change very soon. When I submitted #11 I was planning to cut a release and push it to Rubygems.org, so it's only a matter of minutes... 😄